tests/bifrost/workflow_test.py

```diff
@@ -109,5 +109,6 @@ def test_simulation_workflow_produces_the_same_data_as_before(
     workflow: sciline.Pipeline,
 ) -> None:
     energy_data = workflow.compute(EnergyData[SampleRun])
+    energy_data.save_hdf5('computed_energy_data_simulated.h5')
```

Comment:
I suppose this was added temporarily to make the reference file and should be removed now?

Reply:
Correct